examples/MagicWeatherCompose/gradle/libs.versions.toml

```diff
@@ -35,3 +35,4 @@ compose-ui-test-manifest = { module = "androidx.compose.ui:ui-test-manifest" }
 
 revenuecat = { module = "com.revenuecat.purchases:purchases", version.ref = "purchases" }
 revenuecat-amazon = { module = "com.revenuecat.purchases:purchases-store-amazon", version.ref = "purchases" }
+revenuecat-debugview = { module = "com.revenuecat.purchases:debugview", version.ref = "purchases" }
```

Comment:
Open to thoughts about the dependency name.

Reply:
I think this is fine. I thought about only `debug` but that might imply other stuff like debug logs or something like that